src/main/java/org/apache/commons/lang3/AnnotationUtils.java

```diff
@@ -69,9 +69,9 @@
          */
         @Override
         protected String getShortClassName(final Class<?> cls) {
-            for (final Class<?> iface : ClassUtils.getAllInterfaces(cls)) {
-                if (Annotation.class.isAssignableFrom(iface)) {
-                    return "@" + iface.getName();
+            for (final Class<?> iFace : ClassUtils.getAllInterfaces(cls)) {
```

Comment:
-1: _iface_ is short for _interface_, not _interFace_. We can't use "interface" since its reserved. No change is needed IMO.